Sources/Core/Client/AutoCancellingSubscription.swift

```diff
@@ -34,8 +34,8 @@ import RxSwift
 ///     }
 /// }
 /// ```
-public final class Subscription {
-    fileprivate static let shared = Subscription()
+public final class AutoCancellingSubscription {
```

Comment:
This is no needed because you moved implementation to LLC.

Reply:
I'm not moving implementation to LLC, we're implementing something different there. We still need this for unreadCounts and similar stuff on Core level.